digitalstrategie/templates/search_results.html

```diff
@@ -1,33 +1,40 @@
 {% extends "base.html" %}
-{% load wagtailcore_tags i18n %}
+{% load wagtailcore_tags i18n contrib_tags %}
 
 {% block meta_tag %}
-    <title>{% trans 'Gemeinsam Digital: Berlin - full-text search' %}</title>
+    <title>{% translate 'Gemeinsam Digital: Berlin - full-text search' %}</title>
 {% endblock %}
 
 {% block content %}
     <div class="layout-grid__area--maincontent">
         <div class="modul-headline">
             <h1 class="title">
-                {% trans 'Full-text search' %}
+                {% translate 'Full-text search' %}
             </h1>
         </div>
         <form action="{% url 'search' %}" method="get">
             <div class="searchform-slot ">
                 <div class="form-group">
-                    <label class="aural" for="q">{% trans 'Search term' %}</label>
+                    <label class="aural" for="q">{% translate 'Search term' %}</label>
                     <div class="searchterm">
                         <div class="input-wrapper">
                             <i class="fas fa-search lens" aria-hidden="true"></i>
                             <input
                               class="form-control search"
                               value="{% if request.GET.q %}{{ request.GET.q }}{% endif %}"
-                              placeholder="{% trans 'Search term' %}"
+                              placeholder="{% translate 'Search term' %}"
                               name="q"
-                            >
+                            />
+                            <button class="button button--clean button--clear-input" type="button" title="{% translate 'Empty field' %}">
```

Comment:
More of a thought: do we want to always show this? I somehow would expect it to only be there if you actually searched for something.

Reply:
@goapunk Yeah, you are right. But the example also always has it. So, let's see if someone complains. :)